Core/Frameworks/BaikalAdmin/Controller/Install/VersionUpgrade.php

```diff
@@ -99,7 +99,7 @@ protected function upgrade($databaseConfig, $sVersionFrom, $sVersionTo) {
         if (version_compare($sVersionFrom, '0.3.0', '<')) {
             // Upgrading from sabre/dav 1.8 schema to 3.1 schema.
 
-            if ($databaseConfig['mysql'] === true) {
+            if ($databaseConfig['backend'] == 'mysql') {
```

Comment:
This is the migration for Baikal 0.3.0. That version still has the mysql variable in its config. In this line, we should keep using the old `mysql` switch instead of `backend`

Reply:
oh yeah! My bad, fixed in 35a3d01